src/controllers/authentication.controller.js

```diff
@@ -26,6 +26,10 @@ export default class AuthenticationController {
   static async signUp(req, res) {
     try {
       const payload = await getFormData(req.body);
+      const checkEmail = await findUserByEmail(payload.email);
+      if (checkEmail) {
+        return errorResponse(res, 409, 'user with the provided email exist');
```

Comment:
can you use the `../utils/customMessages.js` and `../utils/statusCodes.js`. You can just move the status code and error message there and call it here.